examples/skybox/main.rs

```diff
@@ -130,6 +130,7 @@ impl framework::Example for Skybox {
             lod_min_clamp: 0.0,
             lod_max_clamp: 100.0,
             compare: wgpu::CompareFunction::Undefined,
+            anisotropy_clamp: 1,
```

Comment:
I think we should change most (if not all) sampler descriptors to end with `.. Default::default()`. That would be more future proof, match the web better, and avoid exposing stuff that is not relevant

Reply:
I agree. Should I change this in this PR? Or should that be separate?

Reply:
let's have this merged, the PR just continues the convention of the old code, it's not required to change that